### ISSUE TITLE
Keras.NET Run will run with Python 3.7 with minor edits.

### ISSUE THREAD

**Issue by Nodice** [SciSharp/Keras.NET#80]
Change pyversion to 37 and it will execute the setup script.

This is important as Anaconda3 now installs 3.7 by default. 

Also runs Tensorflow 1.15 fine.

**Comment by deepakkumar1984** [comment]
I will publish the nuget for py37 version

**Comment by Nodice** [comment]
I spoke too soon on 1.15.

for some reason installing tensorflowjs causing it to stop working when running a basic fit. Dies when running the Mnist_CNN example with ": tf.gradients is not supported when eager execution is enabled. Use tf.GradientTape instead"

A call to: tf.compat.v1.disable_eager_execution() should fix it, but would only be needed running 1.15

**Comment by Nodice** [comment]
Ok I was able to resolve the 1.15 + tensorflowjs tf.gradient issue by disabling eagerexecution.

Add to Backend : Base, then call it.

public static void DisableEager()
        {

            Instance.tensorflow.compat.v1.disable_eager_execution();
        }

**Comment by deepakkumar1984** [comment]
Thanks. Can you pr this change

**Comment by Nodice** [comment]
Done.
